roles/openshift_aws/tasks/elb.yml

```diff
@@ -29,9 +29,9 @@
                    if 'master' in openshift_aws_node_group_type or 'infra' in openshift_aws_node_group_type
                    else openshift_aws_elb_listeners }}"
 
-- name: "Create ELB {{ openshift_aws_elb_name }}"
+- name: "Create ELB {{ l_openshift_aws_elb_name }}"
   ec2_elb_lb:
-    name: "{{ openshift_aws_elb_name }}"
+    name: "{{ l_openshift_aws_elb_name }}"
```

Comment:
Why this change here?

I think this should be `openshift_aws_elb_name`.  The `elb.yml` can be called with `tasks_from` and would require the user to create `l_openshift_aws_elb_name`.